frontend/heart-code/index.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" href="src/HeartIcon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>HeartCode.ai</title>
```

Comment:
misschien hier HeartCode.AI
ai met hoofdletters?
is niet nodig maar ik denk dat her er wel iets leuker uitziet

Reply:
laat maar trouwens, is niet consistent met logo